cmake/SetupSpheral.cmake

```diff
@@ -153,6 +165,7 @@ endif()
 # Build C++ tests and install tests to install directory
 #-------------------------------------------------------------------------------
 if (ENABLE_TESTS)
+  add_subdirectory(${SPHERAL_ROOT_DIR}/tests)
   add_subdirectory(${SPHERAL_ROOT_DIR}/tests/unit/CXXTests)
 
   # A macro to preserve directory structure when installing files
```

Comment:
Do we want the `install_with_directory` command to also find the `tests/cpp` or should those be left out of that command?

Reply:
Should we address this in #304

Reply:
Sure that works for me.